docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
```

Comment:
Committing this file will screw up people's git

Because most people will create this file on their own. Thereafter  they will not be able to pull new changes without conflict